src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -169,8 +169,8 @@ internal static Assembly LoadGeneratedAssembly(Type type, string defaultNamespac
             }*/
 
             // check to see if we loading explicit pre-generated assembly
-            IEnumerable<Attribute> attrs = type.GetTypeInfo().GetCustomAttributes(typeof(XmlSerializerAssemblyAttribute), false);
-            if (attrs.Count() == 0)
+            object[] attrs = type.GetCustomAttributes(typeof(XmlSerializerAssemblyAttribute), false);
+            if (attrs.Length == 0)
             {
                 // Guess serializer name: if parent assembly signed use strong name 
                 AssemblyName name = type.GetTypeInfo().Assembly.GetName();
```

Comment:
>type.GetTypeInfo() [](start = 36, length = 18)

not suggesting cleaning these up now, probably not worth even a tracking issue, but according to code flow there are still 64 cases where the code does type.GEtTypeInfo().